recipe_scrapers/americastestkitchen.py

```diff
@@ -0,0 +1,101 @@
+# mypy: allow-untyped-defs
+
+import functools
+import json
+import re
+
+from recipe_scrapers._grouping_utils import IngredientGroup
+
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, normalize_string
+
+
+class AmericasTestKitchen(AbstractScraper):
+
+    @classmethod
+    def host(cls, domain="americastestkitchen.com"):
+        return domain
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def description(self):
+        return self.schema.description()
+
+    def total_time(self):
+        return get_minutes(self._get_additional_details.get("recipeTimeNote"))
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        ingredients = []
+        for group in self._get_additional_details.get("ingredientGroups"):
+            ingredients += group["fields"]["recipeIngredientItems"]
+        return [self._parse_ingredient_item(i) for i in ingredients]
+
+    def ingredient_groups(self):
+        ingredient_groups = []
+        for group in self._get_additional_details.get("ingredientGroups"):
+            ingredients = [
+                self._parse_ingredient_item(i)
+                for i in group["fields"]["recipeIngredientItems"]
+            ]
+            purpose = group["fields"]["title"]
+            ingredient_groups.append(
+                IngredientGroup(ingredients=ingredients, purpose=purpose)
+            )
+        return ingredient_groups
+
+    def instructions(self):  # add headnote
+        if headnote := self._get_additional_details.get("headnote"):
+            # Ideally this would use HTMLTagStripperPlugin, but I'm not sure how to invoke it here
+            headnote = f"Note: {normalize_string(re.sub(r'<.*?>', '', headnote))}\n"
+        else:
+            headnote = ""
+        return headnote + self.schema.instructions()
+
+    def instructions_group(self):
+        if headnote := self._get_additional_details.get("headnote"):
+            # Ideally this would use HTMLTagStripperPlugin, but I'm not sure how to invoke it here
+            headnote = f"Note: {normalize_string(re.sub(r'<.*?>', '', headnote))}\n"
+        else:
+            headnote = ""
+        return "\n".join(
+            +[
```

Comment:
Possibly missing `[headnote]` or similar here?

Reply:
Ugh, good catch.

Reply:
It seems like useful information, so I don't think we should delete it - but how/where to provide it is another question.

In the test example, if the ragu-preparation and bechamel-preparation tasks were being carried out by two (or more) different people, then it'd be useful for them to know the relevant info.